utilities/user_utilities.py

```diff
@@ -395,6 +407,15 @@ def publish_user_event(eventId):
             # if successful, change status of event to approved.
             else:
                 platform_event_id = result.json()['id']
+                # post eventid to group building block
+                url = "%sint/group/%s/events" % (current_app.config['GROUPS_BUILDING_BLOCK_BASE_URL'], event['createdByGroupId'])
+                result = requests.post(url, headers={"Content-Type": "application/json", "INTERNAL-API-KEY": current_app.config['INTERNAL_API_KEY']},
+                                       data=json.dumps({"event_id": platform_event_id, "creator":{"email": session['email'], "name": session['name'], "user_id": session['uin']} }))
+                if result.status_code != 200:
+                    flash('An error occurred when registering this event with the selected Group. Please contact an administrator to resolve this issue.')
```

Comment:
I tested this part by changing the internal API key and saving an event that's being edited. The error message displayed is. Maybe some other exception is being generated. Please fix this as we need to display appropriate error messages for different error states. Thanks.
<img width="1095" alt="image" src="https://user-images.githubusercontent.com/4306604/192888775-50bdeb27-e277-47c7-9898-5f46b573c5aa.png">

Reply:
@sandeep-ps It checked the permission first. So the code does not reach here. that is why the error message does not send out.

Reply:
Hi, Bing, Thanks for checking this. Could you please make code changes so that the error message gets displayed if there is an error when communicating with the Groups BB? It will be good if we can display the correct error message when there is an issue. Thanks.

Reply:
I am confused about your comment. the internal key is incorrect, so we display what error message?

Reply:
Bing created a follow-up task to investigate displaying the relevant error message [here](https://github.com/rokwire/events-manager/issues/964). I will resolve this as future work will address the issue reported in this conversation. Thanks.

Reply:
ok, thanks.